### DISCUSSION TITLE
Where information is taken from Kibana?

### DISCUSSION THREAD

**Discussion by rocco-96** [telekom-security/tpotce#796] [General]
I was faced with this problem.
I am interested in collecting all the information that is shown by the various honeypots / tools present in kibana.
I know where the daily json files of each honeypot related to the captured data are stored.
My problem concerns the retrieval of the data shown by the various honeypots / tools present in kibana.
I did not understand if there is a dbms to support it (so that the various honeypots query the database) and, if there was, if there was a way to query the database and get that data.

**Comment by kdeiss** [comment]
I also would apreciate if the authors could provide informations regarding this.

**Comment by Tango9191** [comment]
I guess this would be the right place to start:
https://github.com/telekom-security/tpotce/wiki/Reconfigure-logstash.conf
